src/test/scala/io/iohk/ethereum/blockchain/sync/PivotBlockSelectorSpec.scala

```diff
@@ -110,6 +110,52 @@ class PivotBlockSelectorSpec
     )
   }
 
+  it should "retry if there are no enough eligible peers" in new TestSetup {
```

Comment:
can we somehow show that there is no enough eligible peers (assert, name etc)? it is hard for me to figure out that :)